mswindows/osgeo4w/package.sh

```diff
@@ -128,7 +128,11 @@ DLLS="
 	/mingw64/bin/libpcre-1.dll
 	/mingw64/bin/libstdc++-6.dll
 	/mingw64/bin/libgcc_s_seh-1.dll
+    /mingw64/bin/libgcc_s_dw2-1.dll
```

Comment:
indendation correct?

Reply:
good catch. 
Actually, `libgcc_dw2` seems to be 32bit and the 64bit equivalent is `libgcc_s_seh`, so I removed that line. But indentation was inconsistent also other places...

A test build based on OSGeo4W V2 is running here:
https://github.com/ninsbl/OSGeo4W/runs/3806467054?check_suite_focus=true